.github/dependabot.yml

```diff
@@ -341,3 +341,13 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+- package_manager: "javascript"
+  directory: "/samples/snippets"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package_manager: "docker"
+  directory: "/samples/snippets"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
```

Comment:
Will dependabot look for project files in nested directories?
If so, this whole file can be shrinked.